gateway/src/api/latest.rs

```diff
@@ -84,6 +84,14 @@ impl StatusResponse {
     }
 }
 
+#[derive(Debug, Clone, Copy, Deserialize, IntoParams)]
+struct PaginationDetails {
```

Comment:
Seeing it again here makes me think this can be as well something declared under `shuttle-common`. What do you think?

Reply:
Yeah sounds good. We'll look for a good place to put it and remove the duplication.